sdk/search/test-resources.bicep

```diff
@@ -1,22 +1,38 @@
-param searchEndpointSuffix string = 'search.windows.net'
-param storageEndpointSuffix string = 'core.windows.net'
-param location string = '${resourceGroup().location}'
+param environmentEndPoints object = {
+  AzureCloud:{
+    searchEndpointSuffix: 'search.windows.net'
+  }
+  AzureChinaCloud:{
+    searchEndpointSuffix: 'search.chinacloudapi.cn'
```

Comment:
Do we have these values wrong in our config? https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/TestResources/clouds/AzureChinaCloud.json#L11

Reply:
@benbp I picked up the value from the [Azure China developer guide
](https://docs.microsoft.com/en-us/azure/china/resources-developer-guide). Cognitive Search did not have a specific map, so I went with the Azure(in general) *.windows.net -> *.chinacloudapi.cn .